### PR TITLE
PR for @types/react-base-utility type declaration package

### PR DESCRIPTION
**Pull request by AmeyKhoje**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/react-base-utility/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// Type definitions for react-base-utility 1.0
+// Project: https://github.com/AmeyKhoje/react-base-utility#readme
+// Definitions by: Amey Khoje <https://github.com/me>
```
Comment (DangerBotOSS) [inline types/react-base-utility/index.d.ts:3]: - :no_entry_sign: This line should have your github username in it, not /me

Comment (AmeyKhoje) [reply types/react-base-utility/index.d.ts]: Done

**Comment by peterblazejewicz** [comment]
@AmeyKhoje thanks for PR!
Can you make sure, you move the test file into the types directory? thanks!

**Comment by AmeyKhoje** [comment]
@peterblazejewicz Moved!

**Comment by peterblazejewicz** [comment]
@AmeyKhoje the native module contains export for TS types (supposed to be created by TS configuration):
https://github.com/AmeyKhoje/react-base-utility/blob/master/package.json#L20
it looks like distribution is missing decoration files. What's the status with TS declarations?

**Comment by AmeyKhoje** [comment]
@peterblazejewicz I was generating types before with rollup. But now i removed that generation from rollup config.

**Comment by peterblazejewicz** [comment]
OK, that make sense. You will have to update package.json on native package to avoid problems with automatic types acquisition in IDEs.
Now, also please remove all comments that came from template, those are just instructions.
When removed, please compare exported symbols from native module and here, those should match

**Comment by AmeyKhoje** [comment]
@peterblazejewicz Resolved all. Please check now

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]
If you're the author of the underlying package, please ship types in the package instead of using DT. Thanks!

**Comment by AmeyKhoje** [comment]
@RyanCavanaugh I need this to be shipped separately with DT instead adding in package. There is requirement for it.

**Pull request closed without merging**